spec/client_spec.js

```diff
@@ -173,6 +231,33 @@ describe('Client', function() {
 
     });
 
+    describe('#get', function() {
+      beforeEach(function() {
+        sandbox.spy(window, 'setTimeout');
+      });
+
+      it('returns a promise', function() {
+        expect(subject.get('ticket.subject')).to.be.a.promise;
```

Comment:
Can we also test that this promise gets resolved when the expected message is received by the handler?